Flux 2.0 GA Announcement (#1570)
fluxcd/website: content/en/blog/2023-07-17-june-update/index.md

---
author: somtochiama
date: 2023-07-18 20:30:00+00:00

> title: June 2023 Update

Reviewer: ```suggestion
title: Announcing Flux 2.0 GA
```

description: "Flux 2.0 and General Availability!"

> url: /blog/2023/07/june-2023-update/

Reviewer: ```suggestion
url: /blog/2023/07/flux-ga/
```

> tags: [monthly-update]

Reviewer: ```suggestion
tags: [announcement]
```

resources:
- src: "**.{png,jpg}"
  title: "Image #:counter"
---

## Flux 2.0 and General Availability!

![FluxCD community stats](infographic-featured.png)

> Best Buy: Tahir Raza, Staff Engineer - Cloud & Platform Engineering: 
> “GitOps practices and Flux has elevated our engineering: code infra
> as software, eliminate human intervention, accelerate lead time for changes -

> > without compromising security requirements.”

Reviewer: Maybe move this below, as the first phrase is used for social media embeds.

availability of its GitOps components! Flux has continued to grow during its
incredible journey. Its early iteration was built at Weaveworks for their own
needs and for a previous SaaS product built on Flux. Flux led to Weaveworks
CEO Alexis Richardson to coin the term, GitOps, which has taken the world by
storm with a CNCF Working Group, GitOpsCon, GitOps Days, and several GitOps
community groups. Moreover, leaders such as Kubernetes co-creators Brendan Burns
and Joe Beda have stated how GitOps is a natural evolution of Kubernetes itself.

It’s, therefore, no surprise that the open-source Flux project has reached many
great achievements: continued developmental improvements and thought leadership
such as microservices architecture, security-first design, multi-tenancy, OCI and
Cosign support, and more. as part of this journey, Flux is a graduated project in the Cloud
Native Computing Foundation and is the GitOps tool that companies such as Microsoft,
AWS, GitLab, D2iQ, and more trust to deliver GitOps to their customers. 
Enterprises such as SAP, RingCentral, Best Buy, Volvo and many others 
also trust Flux to be an essential part of their Kubernetes journey.

> > Safaricom PLC: Winnie Gakuru, DevSecOps Engineer II: "Safaricom PLC 
> > rovides mobile telephony, mobile money transfer, consumer electronics, e-commerce,
> > cloud computing, data, music streaming, and fiber optic services to the Kenyan
> > Market predominantly and to the wider East Africa. So, Flux has been an essential
> > part of critical areas such as deployment frequency, standardization, and security,
> > among other GitOps capabilities that help us to be competitive. We are excited
> > about Flux 2.0 and the project's continued maturity.

Reviewer: ```suggestion
> Safaricom PLC provides mobile telephony, mobile money transfer, consumer electronics,
> e-commerce, cloud computing, data, music streaming, and fiber optic services to the Kenyan
> Market predominantly and to the wider East Africa. So, Flux has been an essential
> part of critical areas such as deployment frequency, standardization, and security,
> among other GitOps capabilities that help us to be competitive. We are excited
> about Flux 2.0 and the project's continued maturity.
>
> Winnie Gakuru, DevSecOps Engineer II at Safaricom PLC
```

I propose this format for all quotes.

title: Announcing Flux 2.0 GA
description: "Flux 2.0 and General Availability!"
url: /blog/2023/07/flux-ga/
tags: [announcement]
resources:
- src: "**.{png,jpg}"
  title: "Image #:counter"
---

## Flux 2.0 and General Availability!

![FluxCD community stats](infographic-featured.png)

On July 5, 2023, Flux reached a major landmark with Flux 2.0 and the general
availability of its GitOps components! Flux has continued to grow during its
incredible journey. Its early iteration was built at Weaveworks for their own
needs and for a previous SaaS product built on Flux. Flux led to Weaveworks
CEO Alexis Richardson to coin the term, GitOps, which has taken the world by
storm with a CNCF Working Group, GitOpsCon, GitOps Days, and several GitOps
community groups. Moreover, leaders such as Kubernetes co-creators Brendan Burns
and Joe Beda have stated how GitOps is a natural evolution of Kubernetes itself.

> “GitOps practices and Flux has elevated our engineering: code infra
> as software, eliminate human intervention, accelerate lead time for changes -
> without compromising security requirements.”
>
> Tahir Raza, Staff Engineer - Cloud & Platform Engineering at Best Buy

> It’s, therefore, no surprise that the open-source Flux project has reached many
> great achievements: continued developmental improvements and thought leadership
> such as microservices architecture, security-first design, multi-tenancy, OCI and
> Cosign support, and more. as part of this journey, Flux is a graduated project in the Cloud
> Native Computing Foundation and is the GitOps tool that companies such as Microsoft,
> AWS, GitLab, D2iQ, and more trust to deliver GitOps to their customers. 

Reviewer 2: ```suggestion
Intending to be the best GitOps tool available, the Flux project has evolved into a mature and trustworthy software. During its evolution, Flux has accomplished several goals such as low resource consumption by adopting a microservices architecture, safe multi-tenancy through its security-first design and support for bleeding edge innovation by having first-class support for technologies like OCI and Cosign.
We are proud to see that Flux is one of the few CNCF graduated projects and the GitOps tool that companies such as Microsoft, AWS, GitLab, D2iQ, and more trust to deliver GitOps to their customers. 
```

> Now with Flux's GA, we can continue to build the best user experience for our enterprise
> customers on solid foundations."
>
> Viktor Nagy, Senior Product Manager, Environments group at GitLab

### Releases

> “It has been a fantastic journey of rebuilding the original
> Flux into a microservices architecture, adding Flagger as a subproject, getting
> validated as a graduated project in the CNCF, and now reaching GA with Flux 2.0.
> I am grateful to work with great teams, maintainers, contributors, and partners,
> and then  to see major enterprises and cloud providers relying on Flux to start
> or mature their Kubernetes journey. Keeping great company with users (the likes
> of Amazon AWS, D2iQ, Microsoft Azure, VMware, Weaveworks, GitLab, Volvo, SAP,
> Xenit and many more) keeps me motivated for the future innovations and growth for Flux.”
>
> Stefan Prodan, Principal Developer Experience Engineer at Weaveworks, Flux maintainer and
> Flagger creator

Release Cadence: Flux will have at least three minor releases in a year
following the Kubernetes release cadence. The release will happen roughly

> two weeks after the Kubernetes release. Two weeks timeline can be adjusted if more

Reviewer 2: ```suggestion
two weeks after the Kubernetes release. The two weeks timeline can be adjusted if more
```

> two weeks after the Kubernetes release. Two weeks timeline can be adjusted if more

Reviewer 2: ```suggestion
two weeks after a new Kubernetes release. Two weeks timeline can be adjusted if more
```

> time is needed for testing the Kubernetes version.

Reviewer 2: ```suggestion
time is needed for testing compatibility with the new Kubernetes version.
```
